macros/edr/materializations/model/common.sql

```diff
@@ -0,0 +1,17 @@
+{% macro query_row_count_metric() %}
+  {% set query %}
+    select count(*) as row_count
+    from {{ this }}
+  {% endset %}
+  {% do return(elementary.result_value(query)) %}
+{% endmacro %}
+
+{% macro cache_row_count_metric(row_count) %}
```

Comment:
then change this to be `cache_table_metrics`, and iterate over the fields and make a different record for each one (for now it will only be the row count, but it will make it much easier to add more metrics with less queries)